### ISSUE TITLE
[Bug] LocalizedString not displaying in Release mode

### ISSUE THREAD

**Issue by themronion** [xamarin/XamarinCommunityToolkit#1069]
### Description
@maxkoshevoi, i encountered a strange bug. Before the LocalizedString was added to the nuget package i added it to my project and it worked well. But now i've updated to the latest pre, replaced it with the implementation from the nuget and my strings stopped displaying at all (ONLY IN RELEASE MODE). So then, again i copy pasted the LocalizedString from the source code (i assume it is the same as the one currently in the nuget package) and magically it started to work again. 

Any ideas what can be the cause of such a strange behaviour? I'll create a repro if you didnt encounter this problem.

**Comment by maxkoshevoi** [comment]
Are you using 1.1.0-pre1 or pre2? There is a bug in pre1 that is fixed in pre2.

**Comment by maxkoshevoi** [comment]
Oh, it's only in release mode? That might be something new. Thanks for the report I'll take a look at it

**Comment by themronion** [comment]
@maxkoshevoi as i remember i tried both pre-releases. Yeap, the bug is indeed very strange

**Comment by maxkoshevoi** [comment]
Not that strange. I think linker just removed `Localized` property since it thinks that it's not used.

**Comment by themronion** [comment]
Ah, yeah, forgot about the linker. I think so too

**Comment by maxkoshevoi** [comment]
Strange. I cannot reproduce it. Could you please share release configuration of your app? Also, is it Android or iOS?

**Comment by themronion** [comment]
Tried on android only, i am using r8+d8, sdk assemblies only

**Comment by maxkoshevoi** [comment]
Used those settings and everything still works. 
Here's what I've tried: https://github.com/maxkoshevoi/XamarinCommunityToolkit/tree/1069-repro (demo app has `LocalizedString` on settings page)

Could you provide a repro steps / code please?

**Comment by themronion** [comment]
I'll provide a repro then - tomorrow or on monday, thanks for a quick reply

**Comment by themronion** [comment]
@maxkoshevoi  Here u are mate. As we thought, its a linker problem. Adding somewhere in code 
```csharp 
Console.WriteLine(localizedstring.Localized) 
``` 
is a quickfix for now
[LocalizedStringBug.zip](https://github.com/xamarin/XamarinCommunityToolkit/files/6134572/LocalizedStringBug.zip)

**Comment by maxkoshevoi** [comment]
Thanks for the pepro app! Yep, it's linker: 
![image](https://user-images.githubusercontent.com/6609929/111028101-1cea1700-83fd-11eb-82e7-5dc971c43a1e.png)

https://github.com/xamarin/XamarinCommunityToolkit/pull/1071 might fix it, but I was unable to verify it locally. When I created XCT NuGet locally and added it to the app, everything worked even without the fix.

Also I failed to figure out why it fails in your app, but doesn't fail in our demo app for example.
